### ISSUE TITLE
training

### ISSUE THREAD

**Issue by DeekshithSanem** [ruthvika-raj/nanogpt#12]
![Image](https://github.com/user-attachments/assets/17c801fe-98a2-4d9d-98bb-0d3206ef9024)

![Image](https://github.com/user-attachments/assets/3b898340-d2a9-4a35-9079-2ef5c5212d9c)

**Comment by DeekshithSanem** [comment]
trained using GPU acceleration with CUDA. We trained the model for 150 epochs on over 280,000 tokens using a batch size of 512. The initial loss was 6.3 and the final dropped to 0.84. Accuracy improved from 10.5% to 75.3%. Here's a graph of the training loss and accuracy over time.  which took over 12 hours
